test/e2e/k8s_util.go

```diff
@@ -158,6 +158,49 @@ func (k *KubernetesUtils) Probe(ns1, pod1, ns2, pod2 string, port int32, protoco
 	return connectivity, nil
 }
 
+func (k *KubernetesUtils) ProbeEgress(ns, pod, dstName string, port int32, protocol v1.Protocol) (PodConnectivityMark, error) {
```

Comment:
it feels like there is at least some room for unification with the regular `Probe` function